src/controllers/problem.controller.js

```
@@ -91,13 +88,28 @@ const getAll = async (req, res) => {
   }
 
   const key = `problem:${id}`;
 
   try {
+    if (auth) {
+      const token = auth.split(" ")[1];
+      try {
+        const decoded = jwt.verify(token, process.env.TOKEN_SECRET);
+        if (decoded?.sub) {
+          userId = decoded.sub;
+          key = `problems:${limit}:${page}:user:${userId}`;
```

Biome (1.9.4)

[error] Can't assign key because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)